playground/webpack.config.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
              *
              * TODO: define the name of the library here
              */
-            "@": path.resolve(__dirname, "./src/"),
+            "@": path.resolve(__dirname, "../modular-particle-system/src/"),
```

Comment:
you don't need that...use package.json and link your modular particle system as dependency into package.json
try something `npm i ../modular-particle-system`, it will mimic like you install something, but in fact it will use your local package from mono-repo

FYI: Do not forget to add `module`,`main` to package.json of particle-packages

Reply:
This syntax is cool, and I didn't know it existed!

But this means using the compiled library code, rather than including the library compilation in the playground app, right?
Sure, we could still have hot reload by adding a build watch script to the library code, but I'm just thinking isn't this ../ approach better since it does work and without any extra things?